src/App.tsx

```diff
@@ -1,24 +1,97 @@
-/* eslint-disable max-len */
 /* eslint-disable jsx-a11y/control-has-associated-label */
-import React from 'react';
-import { UserWarning } from './UserWarning';
+import React, { useEffect, useState } from 'react';
+import { Header } from './components/Header';
+import { Main } from './components/Main';
+import { Footer } from './components/Footer';
+import { Notification } from './components/Notification';
 
-const USER_ID = 0;
+import { Todo } from './types/Todo';
+import { Error } from './types/Error';
+import { getTodos } from './api/todos';
+import { Loader } from './components/Loader';
+import { formatTodos } from './utils/formatResponse';
 
 export const App: React.FC = () => {
-  if (!USER_ID) {
```

Comment:
You should check if USER_ID is provided, as it was done in the original code. If not, render the UserWarning component.

Reply:
ok